filament/backend/src/opengl/platforms/PlatformEGL.cpp

```diff
@@ -110,36 +116,24 @@ Driver* PlatformEGL::createDriver(void* sharedContext, const Platform::DriverCon
     eglCreateImageKHR = (PFNEGLCREATEIMAGEKHRPROC) eglGetProcAddress("eglCreateImageKHR");
     eglDestroyImageKHR = (PFNEGLDESTROYIMAGEKHRPROC) eglGetProcAddress("eglDestroyImageKHR");
 
-    EGLint configsCount;
-    EGLint configAttribs[] = {
-            EGL_RENDERABLE_TYPE, EGL_OPENGL_ES3_BIT_KHR,        //  0
-            EGL_RED_SIZE,    8,                                 //  2
-            EGL_GREEN_SIZE,  8,                                 //  4
-            EGL_BLUE_SIZE,   8,                                 //  6
-            EGL_ALPHA_SIZE,  8,                                 //  8
-            EGL_DEPTH_SIZE, 24,                                 // 10
-            EGL_RECORDABLE_ANDROID, 1,                          // 12
-            EGL_NONE                                            // 14
-    };
-
-    EGLint contextAttribs[] = {
-            EGL_CONTEXT_CLIENT_VERSION, 3,
-            EGL_NONE, EGL_NONE, // reserved for EGL_CONTEXT_OPENGL_NO_ERROR_KHR below
-            EGL_NONE
-    };
-
-    EGLint pbufferAttribs[] = {
+    EGLint const pbufferAttribs[] = {
```

Comment:
why not make this a Config for consistency?

Reply:
it didn't need it because it's `const`. trying to avoid some heap allocations.

Reply:
ack